### PR TITLE
New WebHooks notifications

### PR DESCRIPTION
**Pull request by mcaramma**

### CODE REVIEW

File: src/administration/integrations/notifications.md [hunk 1]
```diff
@@ -58,3 +58,27 @@ platform integration:add --type health.pagerduty --routing-key YOUR_ROUTING_KEY
...
+
+To do so, register a `health.webhook` integration as follows:
+
+```bash
+platform integration:add --type health.webhook --url=A-URL-THAT-CAN-RECEIVE-THE-POSTED-JSON
+```
+
+Any notification will now be posted to the health.webhook URL.
+
+This integration also allow to specify an optional `shared-key` which will add a `X-JWS-Signature` request header containing the JSON Web Token Signature in JWS Compact Serialization with Unencoded Detached Payload (RFC7797).
```
[the remark this answers is not in the document]
Comment (chadwcarlson) [reply src/administration/integrations/notifications.md]: Maybe also include a link to this option, i.e. "Detached Payload ([RFC7797](https://tools.ietf.org/html/rfc7797))"

[the remark this answers is not in the document]
Comment (chadwcarlson) [reply src/administration/integrations/notifications.md]: And do we need to provide any information - even if it's just a link - as to how to get `X-JWS-Signature`?

File: src/administration/integrations/notifications.md [hunk 2]
```diff
@@ -58,3 +58,50 @@ platform integration:add --type health.pagerduty --routing-key YOUR_ROUTING_KEY
...
+
+The signature is calculated using the given `shared-key` and the fixed header:
+
+```json
+{"alg":"HS256","b64":false,"crit":["b64"]}
+```
+
+The signature permit to verify whether notifications requests to the web endpoint are authentic.
+
+Signature verification is a simple 2 steps process:
```
Comment (chadwcarlson) [inline src/administration/integrations/notifications.md]: s/steps/step

File: src/administration/integrations/notifications.md [hunk 3]
```diff
@@ -58,3 +58,50 @@ platform integration:add --type health.pagerduty --routing-key YOUR_ROUTING_KEY
...
+platform integration:add --type health.webhook --url=A-URL-THAT-CAN-RECEIVE-THE-POSTED-JSON --shared-key JWS-SYMMETRIC-KEY
+```
+
+The signature is calculated using the given `shared-key` and the fixed header:
+
+```json
+{"alg":"HS256","b64":false,"crit":["b64"]}
+```
+
+The signature permit to verify whether notifications requests to the web endpoint are authentic.
```
Comment (chadwcarlson) [inline src/administration/integrations/notifications.md]: The signature permits you/permits the application to verify ... ?

File: src/administration/integrations/notifications.md [hunk 4]
```diff
@@ -58,3 +58,50 @@ platform integration:add --type health.pagerduty --routing-key YOUR_ROUTING_KEY
...
+
+Signature verification is a simple 2 steps process:
+
+```python
+# 1. compute JWS Compact Serialization with Unencoded Detached Payload
```
Comment (chadwcarlson) [inline src/administration/integrations/notifications.md]: s/compute/Compute

File: src/administration/integrations/notifications.md [hunk 5]
```diff
@@ -58,3 +58,60 @@ platform integration:add --type health.pagerduty --routing-key YOUR_ROUTING_KEY
...
...on": {"license_uri": "http://admin.local.c-g.io/_", "plan": "standard", "environments": 10, "storage": 5120, "included_users": 3, "subscription_management_uri": "http://subscription-management", "restricted": true, "suspended": false, "user_licenses": 1}, "repository": {"url": "test_project@git.local.c-g.io/test_project.git", "client_ssh_key": "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDKaBW95l6J3sWZROA4T+pl9Fv4gHUIHtW0R+yqOH/4qzOlWOxUmg5r4V6xHPVmCA+RuXjzmsTy0cJHDVxjxtkjB6g8PXp9G86P9MiZYPWuolpSR/2+ix7mo2zuJEsZQbfGoOaZKQgGvdWNHZusJCy8Ay53CkBMVmlXB+f9Q0U3C1iO/R3EJxdwEBFTx97xzAaBn8hBH1W3SEID/hBlVJiImuGi7CccAtNfyvAaA+VlgSwsX9DDsKFSCHe9q7xckrN0CSp9A/XnA/UyiSiplefjTr0dIb/Xf/aqcML1WomnggopBFk06LkMg31xLiACvlqMArSLd9BP34IanW3oKWjn test_project@platformsh"}, "owner": "alice", "default_domain": null, "status": {"code": "provisioned", "message": "ok"}}, "event": {"time": "2019-07-16T23:13:12.099235+00:00", "origin": {"project": "test_project", "environment": "master-7rqtwti", "service": "mysql", "instance": "0"}, "component": "disk", "subcomponent": "persistent", "severity": "warning", "message": "less than 214.748 MB available", "metrics": {"total:byte": 1073741824, "available:byte": 134217728}}}
```
Comment (ralt) [inline src/administration/integrations/notifications.md]: I'd put:

```
{...request body...}
```

Comment (mcaramma) [reply src/administration/integrations/notifications.md]: Done

**Pull request merged**